templates/taxi/car_list.html

```diff
@@ -10,6 +11,7 @@ <h1>Car list</h1>
           {{ car.model }} ({{ car.manufacturer.name }})
         </li>
       {% endfor %}
+      <button onclick="window.location.href='{% url "taxi:cars-create" %}'">Clic to add a new car!</button>
```

Comment:
You dont have this endpoint `cars-create` :)👀 
And maybe you need to change button position to the top
```suggestion
      <button onclick="window.location.href='{% url "taxi:car-create" %}'">Clic to add a new car!</button>
```